packages/components/organisms/f-content-cards/README.md

```diff
@@ -122,108 +122,108 @@ The following rudimentary example can be used as a guide for implementing this c
 </html>
 ```
 
+
+## Slots
+
+The `ContentCards` component makes heavy use of slots in order to surface different states of data retrieval
+
+### `loading`
+
+The `loading` slot can used by the consuming code to surface a loading state to the user before card data has
```

Comment:
"slot can be used" or "slot is used"

Reply:
@dandel10n IMO, "can be used", as it's not compulsory to supply content for the loading slot.

Reply:
At the moment it says "slot can used"

Reply:
🤦‍♂️ There goes my brain, inserting words again. Thanks for pointing that out, will fix.